src/dashboard/src/requirements/base.txt

```diff
@@ -9,7 +9,7 @@ logutils==0.3.3
 django-tastypie==0.13.2
 django-extensions==1.1.1
 elasticsearch>=1.0.0,<2.0.0
-git+https://github.com/artefactual/archivematica-fpr-admin.git@v1.7.12#egg=archivematica-fpr-admin
+git+https://github.com/artefactual/archivematica-fpr-admin.git@dev/issue-223-nailgun-ng-compat-fix#egg=archivematica-fpr-admin
```

Comment:
I will update once we make the fpr-admin release..

Reply:
Accepted this PR, so you can update!